app/gui/src/controller/searcher/component.rs

```diff
@@ -199,6 +230,23 @@ impl Component {
             _ => false,
         }
     }
+
+    /// Return an optional iterator over the component's aliases from the "ALIAS" tags in the
+    /// entry's documentation. Returns `None` if the entry does not contain documentation.
+    pub fn aliases(&self) -> Option<impl Iterator<Item = &str>> {
+        match &self.data {
+            Data::FromDatabase { entry, .. } => {
+                let aliases = entry.documentation.iter().filter_map(|doc| match doc {
+                    DocSection::Tag { name, body }
+                        if name == ast::constants::ALIAS_DOC_SECTION_TAG_NAME =>
+                        Some(body.as_str().split(',').map(|s| s.trim())),
+                    _ => None,
+                });
+                Some(aliases.flatten())
+            }
+            _ => None,
+        }
+    }
```

Comment:
We could flatten the result of the match, so the function won't need to return an Option anymore (what, I think, is a better API).